nodebuilder/settings.go

```diff
@@ -121,16 +124,19 @@ func initializeMetrics(
 		metric.WithResource(resource.NewWithAttributes(
 			semconv.SchemaURL,
 			semconv.ServiceNameKey.String(fmt.Sprintf("Celestia-%s", nodeType.String())),
-			// TODO(@Wondertan): Versioning: semconv.ServiceVersionKey
 			semconv.ServiceInstanceIDKey.String(peerID.String()),
+			// custom key-val pairs
+			attribute.String("service.buildTime", buildInfo.BuildTime),
+			attribute.String("service.lastCommit", buildInfo.LastCommit),
+			attribute.String("service.semanticVersion", buildInfo.SemanticVersion),
+			attribute.String("service.systemVersion", buildInfo.SystemVersion),
+			attribute.String("service.goVersion", buildInfo.GolangVersion),
```

Comment:
Let's follow semconv keys for such things.

Reply:
unfortunately, no

Reply:
Ok, if there are no conventions for each key we use, let's stick to a custom format for all instead of a mix for consistency. Basically, like you did